client/packages/system/src/Encounter/ListView/columns.ts

```diff
@@ -66,9 +66,8 @@ export const useEncounterListColumns = ({
 
   const columnList: ColumnDescription<EncounterRowFragment>[] = [
     {
-      key: 'encounter-type',
+      key: 'type',
```

Comment:
Any reason why changing it? with 'encounter-type' you might avoid potential name clashes if using a more specific key...

Reply:
`Type` column wasn't sorting cause of invalid `key` value i.e. `encounter-type`. It was expecting the data element key value here. `EncounterRowFragment[]`